source/api/presentation/3.0/index.md

```diff
@@ -1136,6 +1136,10 @@ The following properties are defined by both, and the IIIF representation is mor
 
 The `rights`, `partOf`, and `items` properties are defined by both in the same way.
 
+### 4.8. Keyword Mappings
+
+The JSON-LD keywords `@id`, `@type` and `@none` are mapped to `id`, `type` and `none` by the Presentation API [linked data context][prezi30-ldce]. Thus in content conforming to this version of the Presentation API, the only JSON key beginning with `@` will be `@context`. However, the content may include data conforming to older specifications or external specifications that use keywords beginning with `@`.
+
```

Comment:
...and therefore clients should be prepared to handle both syntaxes?

Reply:
We do say that already in the Extensions section a couple sections above ... but repeating it certainly wouldn't hurt. :+1:

Reply:
"should be prepared to handle" seems too strong, it would seem reasonable to write a v3 and up client that ignored data from earlier specs or other specs that use `@` keywords. Maybe simply "Clients should expect to encounter both syntaxes."?

(We already say "If a client discovers properties that it does not understand, then it MUST ignore them.")